PhysicsTools/NanoAOD/python/triggerObjects_cff.py

```diff
@@ -216,29 +216,42 @@
   selections = selections2016
 )
 
-from PhysicsTools.PatUtils.L1ECALPrefiringWeightProducer_cff import prefiringweight
+from PhysicsTools.PatUtils.L1PrefiringWeightProducer_cff import prefiringweight
 #Next line will be updated once we get UL2016 maps
```

Comment:
This comment could be updated (but ONLY if you need to implement some other change together with it)